src/Theme_Mod_Command.php

```diff
@@ -121,6 +121,52 @@ public function get( $args = array(), $assoc_args = array() ) {
 
 	}
 
+	/**
+	 * Gets all theme mods.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [<mod>...]
+	 * : One or more mods to get.
+	 *
+	 * [--field=<field>]
+	 * : Returns the value of a single field.
+	 *
+	 * [--format=<format>]
+	 * : Render output in a particular format.
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
+	 *   - json
+	 *   - csv
+	 *   - yaml
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Get all theme mods.
+	 *     $ wp theme mod list
+	 *     +------------------+---------+
+	 *     | key              | value   |
+	 *     +------------------+---------+
+	 *     | background_color | dd3333  |
+	 *     | link_color       | #dd9933 |
+	 *     | main_text_color  | #8224e3 |
+	 *     +------------------+---------+
+	 *
+	 */
+	public function list( $args = array(), $assoc_args = array() ) {
+		if ( ! array_key_exists( 'all', $assoc_args ) ) {
+			$assoc_args['all'] = 1;
+		}
+		// If no format was given ,set to default 'table'.
+		if ( ! array_key_exists( 'format', $assoc_args ) ) {
```

Comment:
Setting the default `--format` should be unnecessary, as the default is already provided in https://github.com/wp-cli/extension-command/pull/100/files#diff-03f3f9875c58703b8a8855ca3e5ca77eR138 and will be enforced by the system.